lesson11/main.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const body = document.getElementById("body");
+const wrapper = document.createElement("div");
+body.appendChild(wrapper);
+const startLoading = () => {
+   const loadingIcon = document.createElement("img");
+   loadingIcon.src = "img/loading-circle.gif";
+   wrapper.appendChild(loadingIcon);
+};
+const endLoading = () => wrapper.remove();
+const jsonURL = "https://jsondata.okiba.me/v1/json/KKl45211103113313";
+
+async function fetchData() {
+   try {
+      const response = await fetch(jsonURL);
+      const json = await response.json();
+      return json.data;
+   } catch(error) {
+      throw new Error(`データの取得に失敗しました：${error}`);
+   } finally {
+      console.log('処理が終了しました。');
```

Comment:
nits
I think this is unnecessary, but I would like to know if there is any reason for this.

Reply:
There was no particular meaning there. 
It was just a remnant of a previous lesson.
I deleted that.

c290849